### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**

### CODE REVIEW

**Review by simran61 — APPROVED** [review]
Looks good...clean code

File: src/index.css [hunk 1]
```diff
@@ -11,6 +11,7 @@
   --note-border-dark-clr: #555555;
   --background: var(--light-bg-clr);
   --text-primary: var(--font-clr);
+  --dark-purple-clr: #4c3a51;
```
Comment (SanketDhabarde) [inline src/index.css:14]: You can be consistent with the namings either user var or hex. It will look good

**Review by SanketDhabarde — APPROVED** [review]
- Code looks good
- Good to merge

File: src/index.css (hunk 1 above)
Comment (abhisheksahani-nova) [reply src/index.css:14]: I have use var beacause they are  component library colors which are already available

**Pull request merged**